crates/oxc_linter/src/rules/react/jsx_props_no_spread_multi.rs

```diff
@@ -0,0 +1,100 @@
+use std::collections::HashMap;
+
+use oxc_ast::{ast::JSXAttributeItem, AstKind};
+use oxc_diagnostics::OxcDiagnostic;
+use oxc_macros::declare_oxc_lint;
+use oxc_span::{Atom, Span};
+
+use crate::{context::LintContext, rule::Rule, AstNode};
+
+fn jsx_props_no_spread_multi_diagnostic(span0: Span, span1: Span) -> OxcDiagnostic {
+    OxcDiagnostic::warn("Disallow JSX prop spreading the same identifier multiple times.")
+        .with_help("Remove duplicate spread attributes.")
+        .with_labels([span0, span1])
+}
+
+#[derive(Debug, Default, Clone)]
+pub struct JsxPropsNoSpreadMulti;
+
+declare_oxc_lint!(
+    /// ### What it does
+    /// Enforces that any unique expression is only spread once.
+    /// Generally spreading the same expression twice is an indicator of a mistake since any attribute between the spreads may be overridden when the intent was not to.
+    /// Even when that is not the case this will lead to unnecessary computations being performed.
+    ///
+    /// ### Example
```

Comment:
Please follow the convention described in the docs template when `just create-react-rule` is run

Reply:
I’ve added a section titled ‘Why is this bad?'
https://github.com/oxc-project/oxc/pull/4866/commits/c9f84b41d8e1415022178602467c5be666a0140a